bin/setup_db

```diff
@@ -79,6 +79,7 @@ def setup_db(host, port, name):
         r.db(name).table('users').index_create('user_id').run(conn)
         r.db(name).table("users").index_create("username").run(conn)
         r.db(name).table("users").index_create("email").run(conn)
+        r.db(name).table("users").index_create("name").run(conn)
```

Comment:
I think we may be using this incorrectly, but I will make a ticket for that and look into it later.

Reply:
yeah, I think indexing is only for if we want to use it as a primary key in a search.

Reply:
The index is added to trade memory for disk. If we don't think there will be a performance hit for large sets, then we can safely remove it. Note the query itself will also need to change.

From https://rethinkdb.com/api/javascript/order_by/:

> Sorting without an index requires the server to hold the sequence in memory, and is limited to 100,000 documents